src/test/java/pageObject/test/PageObjectLogin.java

```diff
@@ -0,0 +1,22 @@
+package pageObject.test;
+
+import org.junit.jupiter.api.Test;
+import pageObject.BaseEntities.BaseTest;
+import pageObject.pages.InMailPageFactory;
+import pageObject.pages.LoginPageFactory;
+
+public class PageObjectLogin extends BaseTest {
+
+    @Test
+    void loginWithPageObject() {
+        LoginPageFactory loginPageFactory = new LoginPageFactory(true);
```

Comment:
please read more about PO pattern, what methods it should contain, the reactor your PO and tests.